src/Tribe/REST/V1/Endpoints/Archive_Event.php

```diff
@@ -64,6 +64,28 @@ public function get( WP_REST_Request $request ) {
 			: false;
 		$args['s'] = $request['search'];
 
+		/**
+		 * Allows users to make the REST API return events from a more specific, localized date range.
+		 *
+		 * With this enabled, "inclusive" start dates and end dates are supplied: the first second
+		 * of the specified start date, and the last second of the specified end date. With this
+		 * disabled, the localized but otherwise-unmodified start and dates are used, which may yield fewer results.
+		 *
+		 * @since TBD
+		 *
+		 * @param bool $use_inclusive Defaults to true. Whether to use "inclusive" start and end dates.
+		 */
+		if ( apply_filters( 'tribe_rest_events_use_inclusive_start_end_dates', true ) ) {
```

Comment:
We should probably rejig the name so that the hook is prefixed with `tribe_events_` and `rest` can come after that

Reply:
will do!

Reply:
@barryhughes it seems like the norm for REST-related filters+actions is `tribe_rest_` ... 🤔 i'm going to ask about this on scrum today